### PR TITLE
fix #1152: Removed wrapping of filename

### LINKED ISSUE #1152: Wrap file name for uploader component
**Current component**

* Truncates text. User cannot see the full file name. 
* Figure out what happens with long file names that have no space

----

## Acceptance criteria

**Current implementation where text is truncated**
image-20230502-211710

**Updated design**
image-20230608-165807

### PR DESCRIPTION
**Pull request by ArakTaiRoth**
Also updated a date file for Sidebar Group, as it was failing to compile because of a JSON error.

### CODE REVIEW

File: libs/web-components/src/components/file-upload-card/FileUploadCard.svelte [hunk 1]
```diff
@@ -175,10 +175,9 @@
 
   .filename {
     grid-area: filename;
-
     font-size: var(--goa-font-size-4);
-    white-space: nowrap;
     overflow: hidden;
```
Comment (chrisolsen) [inline libs/web-components/src/components/file-upload-card/FileUploadCard.svelte:179]: Can overflow and text-overflow also be removed?

Comment (ArakTaiRoth) [reply libs/web-components/src/components/file-upload-card/FileUploadCard.svelte]: They can, and have been.

**Review by chrisolsen — APPROVED** [review]

**Pull request merged**